src/js/actions/GuideActions.js

```diff
@@ -1,24 +1,29 @@
-import AppDispatcher from "../dispatcher/AppDispatcher";
-import GuideConstants from "../constants/GuideConstants";
-
-const GuideActions = {
-  /**
-   * @param {String} id to ignore
-   */
-  ignore: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_IGNORE, id
-    });
-  },
-
-  /**
-   * @param {String} id to follow
-   */
-  follow: function (id) {
-    AppDispatcher.dispatch({
-      actionType: GuideConstants.ORG_FOLLOW, id
-    });
-  }
-};
-
-export default GuideActions;
+var Dispatcher = require("../dispatcher/Dispatcher");
+
+var GuideActions = (function (_Dispatcher) {
+  function _GuideActions () { }
```

Comment:
Sorry can you clarify what the module factory is? I was copying the format of other actions file that I had seen in the project.  Unless there is a strong reason for having the anonymous function which accepts the dispatcher, we could just instead do the following:
import Dispatcher from "../dispatcher/Dispatcher";

module.exports = {
  ignore: function (we_vote_id) {
    Dispatcher.loadEndpoint("organizationFollowIgnore", { organization_we_vote_id: we_vote_id} );
  },

  follow: function (we_vote_id) {
    Dispatcher.loadEndpoint("organizationFollow", { organization_we_vote_id: we_vote_id} );
  }
....<other functions>
}